frontend/src/__tests__/cypress/cypress/pages/pipelines/topology.ts

```diff
@@ -122,6 +122,14 @@ class RunDetails extends PipelinesTopology {
     return new DetailsItem(() => cy.findByTestId(`detail-item-${key}`));
   }
 
+  private findStatusLabel(timeout = 10) {
+    return cy.get('[data-testid="status-icon"]', { timeout });
+  }
+
+  expectStatusLabelToBe(statusValue: string, timeout = 180000) {
+    this.findStatusLabel(timeout).should('have.text', statusValue);
+  }
```

Comment:
I don't think we should be having a large timeout by default. Let the individual test cases supply larger timeouts on a per need basis. Omit the timeout to use the default.

```suggestion
  private findStatusLabel(timeout?: number) {
    return cy.get('[data-testid="status-icon"]', { timeout });
  }

  expectStatusLabelToBe(statusValue: string, timeout?: number) {
    this.findStatusLabel(timeout).should('have.text', statusValue);
  }
```